src/components/profile/MyProfile.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import ApptTicker from './ApptTicker';
+import MyBarbers from './MyBarbers';
+import MyCuts from './MyCuts'
+import MyBookmarks from './MyBookmarks';
+
+
+class MyProfile extends Component {
+    state = {
+        myBarbersView: true,
+        myCutsView: false,
+        myBookmarksView: false
+    };
+
+    onMyBarbersPress = () => {
+        if (!this.state.myBarbersView) this.setState({ myBarbersView: true, myCutsView: false, myBookmarksView: false });
+    };
+
+    onMyCutsPress = () => {
+        if (!this.state.myCutsView) this.setState({ myBarbersView: false, myCutsView: true, myBookmarksView: false });
+    };
+
+    onMyBookmarksPress = () => {
+        if (!this.state.myBookmarksView) this.setState({ myBarbersView: false, myCutsView: false, myBookmarksView: true });
+    };
+  
+    renderProfileScreen() {
```

Comment:
can we do something like this in `renderProfileScreen()` instead

```suggestion
    renderProfileScreen() {
    if (this.state.myCutsView) return <MyCuts />;
    if (this.state.myBookmarksView) return <MyBookmarksView />;
    return <MyBarbers />;
    }
```